<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="consumerBarLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="brokerAddressLayout">
        <item>
         <widget class="QLabel" name="brokerAddressLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Broker address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="brokerAddressEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line1_1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="startAtLabel">
        <property name="text">
         <string>Start at</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="startAtSelector"/>
      </item>
      <item>
       <widget class="QLineEdit" name="startOffsetEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="startTimeEdit"/>
      </item>
      <item>
       <widget class="Line" name="line1_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="endAtLabel">
        <property name="text">
         <string>Stop at</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="endAtSelector"/>
      </item>
      <item>
       <widget class="QLineEdit" name="endOffsetEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="endTimeEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeView" name="topicPartitionSourceTree"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2_1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="partitionInfoBox">
          <property name="title">
           <string>Partition info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="partitionInfoLayout">
             <item>
              <widget class="QLabel" name="lowOffsetLabel">
               <property name="text">
                <string>Low offset:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lowOffsetValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line3_1">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="highOffsetLabel">
               <property name="text">
                <string>High offset:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="highOffsetValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line3_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lagLabel">
               <property name="text">
                <string>Current offset:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lagValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Source info</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
